colorservice/Numbermethods.java

```diff
@@ -0,0 +1,28 @@
+package colorservice;
+
+import model.ColorPair;
+import model.MajorColor;
+import model.MinorColor;
+
+public class Numbermethods {
+	
+	public static void testNumberToPair(int pairNumber,
```

Comment:
Production code and test code are in the same file. It is a good approach to keep them separately.
By doing so, we can test production code independently and effectively and also can exclude test code easily during shipment of software to field.